db/migrate/20190604090631_create_external_urls.rb

```diff
@@ -0,0 +1,11 @@
+class CreateExternalUrls < ActiveRecord::Migration[5.0]
+  def change
+    create_table :external_urls, :id => :bigserial do |t|
+      t.references :refers_to, :polymorphic => true, :index => true, :type => :bigint
```

Comment:
if the `index => true` here does't work, lets remove it (it should create `external_urls` for you, but it's not implemented in all migration methods)

Reply:
`refers_to` maybe just `resource` ?